docs/diagrams/AddNote_Sequence.puml

```diff
@@ -0,0 +1,86 @@
+@startuml
```

Comment:
Guys for the UML diagram the everthing seems to flow well,  but the class header in the sequence diagrams needs to have a colon preceding the class name, but I'm not sure if plantUML  by default doesn't insert a colon.